force-app/main/default/lwc/groupServiceDelivery/groupServiceDelivery.js

```diff
@@ -22,7 +22,7 @@ export default class GroupServiceDelivery extends LightningElement {
     fieldSet;
 
     serviceField = SERVICE_FIELD.fieldApiName;
-    fieldSetField = BULK_SERVICE_DELIVERY_FIELD_SET_FIELD.fieldApiName;
+    fieldSetField = SERVICE_DELIVERY_FIELD_SET_FIELD.fieldApiName;
     fieldsToExclude = [
         CONTACT_FIELD.fieldApiName,
         PROGRAM_ENGAGEMENT_FIELD.fieldApiName,
```

Comment:
I'd like to discuss whether we use the CONSTs or store the api names and use those. We are wildly inconsistent. I don't have a strong opinion either way, just that I like consistency.

Reply:
yea I just thought since every time we accessed the field we also used .fieldApiName it just made more sense to store that since we never need access to the entire field here.